.github/workflows/test.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         # matrixed execution for parallel gh-action performance increases
         python_version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, macos-13]
```

Comment:
Moving forward, is it worth specifying multiple specific os rather than defaulting to latest?

Reply:
Great point, thank you! I think this would be a good idea, and aligns with existing practices in this + other Cytomining projects.

Reply:
I created #197 to keep this PR isolated to only MacOS changes.